### ISSUE TITLE
Castle FLag Vs. HUD

### ISSUE THREAD

**Issue by K-Dubs1** [Fluvian/tpp#45]
This one is more of a suggestion, but since the patent castle model is so old could there be a chance that the mario icon wasn't cut off? After all, it didn't have to be condensed down to 16x16 and it was likely drawn outside of a 16x16 space. Since there's pretty much no proof as the footage is so low quality, this is really more of a theory.
<img width="209" alt="Screen Shot 2022-06-11 at 2 38 13 PM" src="https://user-images.githubusercontent.com/77821101/173200926-f74a6cdf-dcac-47a1-9604-6cb49dd08e3d.png">

**Comment by Supergub5871** [comment]
heres proof that this mario head is on the flags in the beta footage

![Screenshot 2022-06-11 154004](https://user-images.githubusercontent.com/90076568/173204202-9b452795-80e8-4f01-861b-010eaffc3c48.png)

**Comment by Supergub5871** [comment]
can you see it?

**Comment by legony64** [comment]
> heres proof that this mario head is on the flags in the beta footage
> 
> ![Screenshot 2022-06-11 154004](https://user-images.githubusercontent.com/90076568/173204202-9b452795-80e8-4f01-861b-010eaffc3c48.png)

Yes, it was in the Spaceworld 95' demo. What K-Dubs meant is that Mario's icon was cropped to 16x16 in the HUD, but it shouldn't be cropped to 16x16 in the flag since the size is bigger.

**Comment by Supergub5871** [comment]
ok

**Comment by Supergub5871** [comment]
came back almost a year later i was so stupid back then
